src/lib/cfa/ntp_conf.rb

```diff
@@ -0,0 +1,248 @@
+require "cfa/base_model"
+require "cfa/augeas_parser"
+require "cfa/matcher"
+
+module CFA
+  # class representings /etc/ntp.conf file model. It provides helper to manipulate
+  # with file. It uses CFA framework and Augeas parser.
+  # @see http://www.rubydoc.info/github/config-files-api/config_files_api/CFA/BaseModel
+  # @see http://www.rubydoc.info/github/config-files-api/config_files_api/CFA/AugeasParser
+  class NtpConf < ::CFA::BaseModel
+    PARSER = CFA::AugeasParser.new("ntp.lns")
+    DEFAULT_PATH = "/etc/ntp.conf".freeze
+    SIMPLE_SETTINGS = ["logfile", "driftfile"].freeze
+    AUTH_COMMANDS = ["trustedkey", "requestkey", "controlkey", "keys"].freeze
+    COMMAND_RECORDS = [
+      "server",
+      "peer",
+      "broadcast",
+      "manycastclient",
+      "multicastclient",
+      "manycastserver"
+    ].freeze
+
+    def self.ntp_attributes(*attrs)
+      attrs.each do |attr|
+        define_method(attr) { get_attribute(attr.to_s) }
+        define_method("#{attr}=") { |record| set_attribute(attr.to_s, record) }
+        define_method("delete_#{attr}") { delete_attribute(attr.to_s) }
+      end
+    end
+
+    ntp_attributes :driftfile, :logfile, :keys, :requestkey, :controlkey
+
+    def initialize(path: DEFAULT_PATH, file_handler: nil)
+      super(PARSER, path, file_handler: file_handler)
+    end
+
+    def servers
+      RecordCollection.new(ServerRecord, data.collection("server"))
+    end
+
+    def peers
+      RecordCollection.new(PeerRecord, data.collection("peer"))
+    end
+
+    def restricts
+      RecordCollection.new(RestrictRecord, data.collection("restrict"))
+    end
+
+    private
+
+    def get_attribute(name)
+      return nil if data[name].nil?
+      AttributeRecord.new_from_augeas(data[name])
+    end
+
+    def set_attribute(name, record)
+      data[name] = record.to_augeas
+    end
+
+    def delete_attribute(name)
+      data.delete(name)
+    end
```

Comment:
well, this smells a bit from code POV. ruby do not use get/set and also all have attribute as common, which indicate that it will probably be better if it is method of attribute object. 
also whole this handling is a bit duplicate to existing API https://github.com/config-files-api/config_files_api/blob/master/lib/cfa/base_model.rb#L97

so, if you need more complex functionality then simple get value/ set value I suggest to have attribute as object like

```
class Attribute # probably merge with AttributeRecord class
  def initialize(name, parent)
    @name = name
    @data = parent
  end

  def exists?
    !!@data[@name]
  end

  def value=(value)
    @data[@name] = value.to_augeast
  end

  ...
end
```